src/Tests/Characters.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Character from '../components/Characters/Characters';
+import store from '../redux/configureStore';
+
+describe('Testing Characters:', () => {
+  it('Should render all characters', () => {
+    const tree = render(
+      <React.StrictMode>
+        <Provider store={store}>
+          <Router>
+            <Character />
+          </Router>
+        </Provider>
+      </React.StrictMode>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
![Testing](https://user-images.githubusercontent.com/93039359/197307042-1f53e6c8-7077-4f5f-8368-0e3d7459d3cd.PNG)

- [ ] Your tests are not passing. Kindly make your application work as expected to pass the tests 😊

Reply:
<img width="848" alt="Screen Shot 2022-10-22 at 9 26 06" src="https://user-images.githubusercontent.com/103044822/197323947-4be2808b-a7a8-4ac8-9eb2-225e0b3004b9.png">